plotly/plotly/plotly.py

```diff
@@ -1199,17 +1194,30 @@ def api_url(cls, resource):
     @classmethod
     def headers(cls):
         credentials = get_credentials()
+
         # todo, validate here?
-        un, api_key = credentials['username'], credentials['api_key']
-        encoded_un_key_pair = base64.b64encode(
-            six.b('{0}:{1}'.format(un, api_key))
-        ).decode('utf8')
+        username, api_key = credentials['username'], credentials['api_key']
+        encoded_api_auth = base64.b64encode(six.b('{0}:{1}'.format(
+            username, api_key))).decode('utf8')
 
-        return {
-            'authorization': 'Basic ' + encoded_un_key_pair,
+        headers = {
+            'authorization': 'Basic ' + encoded_api_auth,
```

Comment:
since this is getting set in the following `if` statement, maybe remove this line and put it in an `else`?